main.js

```diff
@@ -1,49 +1,54 @@
-import tasks from "./tasks.json" assert { type: "json" };
-
 const list = document.getElementById("menu");
-const searchBar = document.getElementById("searchBar");
+let taskList = [];
 
-//list display section
-var renderLists = function (lists) {
+const loadTask = async () => {
+  try {
+    const res = await fetch("http://localhost:3000/tasks");
+    taskList = await res.json();
+    renderLists(taskList);
+  } catch (err) {
+    console.error(err);
+  }
+};
+var renderLists = (lists) => {
   const ItemList = lists
     .map(
       (tasks) =>
-        `<li><a href="${tasks.href}" target="iframes-list"> <h4>${tasks.name}</h4></a></li>`
+        `<li class="list-group-item bg-success mt-1 p-3 "><a class="text-light text-decoration-none" href="${tasks.href}" target="iframes-list"><h4>${tasks.name}</h4></a></li>`
     )
     .join("");
   list.innerHTML = ItemList;
+  const targetBtn = document.getElementsByTagName("a")[0];
+  const clickEvent = new MouseEvent("click", {
+    bubbles: true,
+    cancelable: false,
+  });
+  targetBtn.dispatchEvent(clickEvent);
 };
-renderLists(tasks);
-//task filter section
+loadTask();
 const render = () => {
```

Comment:
This method should be named as setupEventListeners and add all addEventListeners inside it, render  name does not make sense here